sdkservice/sdkservice.go

```diff
@@ -24,6 +30,42 @@ func SdkServiceURL(version string) string {
 	return fmt.Sprintf(sdkServiceURLFmt, version)
 }
 
+type SDKServiceClients map[string]sdkutilities.Client
+
+func (clients SDKServiceClients) GetSDKServiceClient(chainName, version string) (sdkutilities.Client, *apierrors.Error) {
+	if v, ok := sdkExceptionMappings[version]; ok {
+		version = v
+	}
+
+	client, ok := clients[version]
+	if !ok {
+		return client, apierrors.New(
+			"chains",
+			fmt.Sprintf("cannot retrieve sdk-service for version %s with chain name %v", version, chainName),
+			http.StatusBadRequest,
+		).WithLogContext(
+			fmt.Errorf("cannot retrieve chain's sdk-service for version %s", version),
+			"name",
+			chainName,
+		)
+	}
```

Comment:
let's use apierrors from handlers only, so this code can be re-used in other parts of the codebase if we want to

```suggestion
	if !ok {
		return client, fmt.Errorf("cannot retrieve sdk-service client for version %s", version)
	}
```

Reply:
Instead of writing same apierrors.New code in lot of places, I thought we can have it here only once. And also we can get only error text using e.InternalCause. Hope it will not be any problem. WDYT?

Reply:
uhm ok let's leave it here, we can just remove these lines since they don't add anything:
```
.WithLogContext(
			fmt.Errorf("cannot retrieve chain's sdk-service for version %s", version),
			"name",
			chainName,
		)
```